src/bundle/Resources/public/js/scripts/core/custom.dropdown.js

```diff
@@ -52,22 +55,14 @@
 
         clearCurrentSelection() {
             this.sourceInput.querySelectorAll('option').forEach((option) => (option.selected = false));
-            this.itemsContainer.querySelectorAll(SELECTOR_ITEM).forEach((option) => option.classList.remove(CLASS_ITEM_SELECTED));
+            this.itemsContainer.querySelectorAll(SELECTOR_ITEM).forEach((option) => option.classList.remove(CLASS_ITEM_SELECTED_IN_LIST));
             this.container.querySelector(SELECTOR_SELECTION_INFO).innerHTML = '';
         }
 
         hideOptions() {
-            return this.itemsContainer.classList.add(CLASS_ITEMS_HIDDEN);
-        }
-
-        deselectPlaceholderOption() {
-            const placeholderItem = this.itemsContainer.querySelector('[data-value=""]');
-
-            if (!placeholderItem) {
-                return;
-            }
+            doc.body.removeEventListener('click', this.onClickOutside);
 
-            placeholderItem.selected = false;
+            return this.itemsContainer.classList.add(CLASS_ITEMS_HIDDEN);
```

Comment:
Is there a reason behind returning this? I checked what value is returned by `classList.add` and it returns `undefined`.

Reply:
@sunpietro is this something you will address before merge?

Reply:
One way or another, the function will return `undefined`. I see no risk here.